src/util/system.cpp

```diff
@@ -635,6 +635,12 @@ bool HelpRequested(const ArgsManager& args)
     return args.IsArgSet("-?") || args.IsArgSet("-h") || args.IsArgSet("-help") || args.IsArgSet("-help-debug");
 }
 
+void SetupHelpOptions(ArgsManager& args)
+{
+    args.AddArg("-?", "Print this help message and exit", false, OptionsCategory::OPTIONS);
```

Comment:
Have you considered doing this on `ArgsManager` constructor?

Reply:
I'd prefer not to bloat the constructor with stuff, since we've had countless initialization and memory issues due to that.

Reply:
Fair, you can resolve this.